server/server.js

```diff
@@ -17,7 +17,7 @@ export default function Server(connector, options = {}) {
   app.post("/notify", NotifyHandler);
 
   // Error Handler
-  app.use((err, req, res, next) => { // eslint-disable-line no-unused-vars
+  app.use((err, req, res) => { // eslint-disable-line no-unused-vars
```

Comment:
yes, but we would need to verify if the `computeHandler` has proper error handling